src/analysis/normed_space/continuous_affine_map.lean

```diff
@@ -70,4 +72,107 @@ begin
   exact (f : P →ᵃ[R] Q).linear_eq_zero_iff_exists_const,
 end
 
+@[simp] lemma to_affine_map_cont_linear (f : V →L[R] W) :
+  f.to_continuous_affine_map.cont_linear = f :=
+by { ext, refl, }
+
+@[simp] lemma zero_cont_linear :
+  (0 : P →A[R] W).cont_linear = 0 :=
+rfl
+
+@[simp] lemma add_cont_linear (f g : P →A[R] W) :
+  (f + g).cont_linear = f.cont_linear + g.cont_linear :=
+rfl
+
+@[simp] lemma sub_cont_linear (f g : P →A[R] W) :
+  (f - g).cont_linear = f.cont_linear - g.cont_linear :=
+rfl
+
+@[simp] lemma neg_cont_linear (f : P →A[R] W) :
+  (-f).cont_linear = -f.cont_linear :=
+rfl
+
+@[simp] lemma smul_cont_linear (t : R) (f : P →A[R] W) :
+  (t • f).cont_linear = t • f.cont_linear :=
+rfl
+
+lemma decomp (f : V →A[R] W) :
+  (f : V → W) = f.cont_linear + function.const V (f 0) :=
+begin
+  rcases f with ⟨f, h⟩,
+  rw [coe_mk_const_linear_eq_linear, coe_mk, f.decomp, pi.add_apply, linear_map.map_zero, zero_add],
+end
+
+section normed_space_structure
+
+variables (f : V →A[𝕜] W)
+
+noncomputable instance : has_norm (V →A[𝕜] W) := ⟨λ f, max ∥f 0∥ ∥f.cont_linear∥⟩
```

Comment:
can you put a warning somewhere that this norm does not satisfy `||f o g|| \<= ||f|| ||g||`?